### PR TITLE
Added libraries

### PR DESCRIPTION
**Pull request by sahiljain112**

### CODE REVIEW

File: registry.json [hunk 1]
```diff
@@ -371,6 +373,7 @@
   "jsnlog": "npm:jsnlog",
   "jsol": "github:daepark/JSOL",
   "jsplumb": "github:sporritt/jsPlumb",
+  "jspm-loader-css":"npm:jspm-loader-css",
```
Comment (guybedford) [inline registry.json]: I'm not sure this is still supported? Is it still working in jspm 0.16? I'm not sure it does in jspm 0.17?

Comment (sahiljain112) [inline registry.json]: Yes. You're right. I switched to systemjs/plugin-css for loading css.

Comment (guybedford) [inline registry.json]: Ok thanks, it may be best to remove this then.

Comment (aluanhaddad) [inline registry.json]: Otherwise this looks great.

**Pull request merged**